pkg/apis/openebs.io/v1alpha1/cstor_pool.go

```diff
@@ -53,6 +53,7 @@ type CStorPoolAttr struct {
 	OverProvisioning bool   `json:"overProvisioning"` //true or false
 }
 
+// CStorPoolPhase is a CStorPool phase.
```

Comment:
Please put it like this:
`CStorPoolPhase is a typed string for phase field of CStorPool`

Reply:
@sonasingh46 done